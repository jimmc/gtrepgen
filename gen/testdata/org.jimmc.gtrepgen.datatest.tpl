Data test {{.}}
Row: {{with row .}}
  a: {{.a}}
  b: {{.b}}
  c: {{.c}}
{{- end}}
Rows: {{range rows .}}
  a: {{.a}}
  b: {{.b}}
  c: {{.c}}
{{- end}}
