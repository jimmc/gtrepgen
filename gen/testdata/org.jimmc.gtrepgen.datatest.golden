Data test top
Row: 
  a: 1
  b: 2.2
  c: Three:top
Rows: 
  a: 1
  b: 2.2
  c: Three:top
  a: 11
  b: 12.2
  c: Thirteen:top
  a: 21
  b: 22.2
  c: Twentythree:top
